examples/wasm_exec.js

```diff
@@ -0,0 +1,535 @@
+// Copyright 2018 The Go Authors. All rights reserved.
```

Comment:
Can you put this in a `examples/vendor` folder maybe? Just so people know it isn't our code (and maybe include the SHA1 you got it from)